meta-luneos/recipes-webos-owo/luna-sysmgr/luna-sysmgr.bb

```diff
@@ -53,6 +56,12 @@ do_install:append() {
         install -v -m 644 ${S}/conf/luna.conf ${D}${webos_sysconfdir}
     fi
 
+    # Apply configured boot applications
+    if [ -f ${D}${webos_sysconfdir}/luna.conf ]
+    then
+        sed -i "1,\$s/Applications=org.webosports.app.phone;com.palm.app.email;com.palm.app.calendar/Applications=${LUNEOS_BOOT_APPS}/" ${D}${webos_sysconfdir}/luna.conf
```

Comment:
Minor remark: I would have put 
`s/Applications=.*/Applications=${LUNEOS_BOOT_APPS}/`
instead, to reflect the fact that we want to override any existing value

Reply:
Fair. I'd need to make this match the section, then, so don't take until I figure that out